flink-runtime/src/main/scala/org/apache/flink/runtime/akka/AkkaUtils.scala

```diff
@@ -291,12 +291,22 @@ object AkkaUtils {
     ConfigFactory.parseString(config)
   }
 
-  def getThreadPoolExecutorConfig: Config = {
+  def getThreadPoolExecutorConfig(configuration: Configuration): Config = {
+    val priority = configuration.getInteger(MetricOptions.QUERY_SERVICE_THREAD_PRIORITY)
```

Comment:
I would prefer to pass in the the thread priority as a parameter of the `getThreadPoolExecutorConfig` because otherwise we couple all `ThreadPoolExecutors` to the `MetricOptions.QUERY_SERVICE_THREAD_PRIORITY`.